amp/WEB-INF/src/org/digijava/module/aim/dbentity/AmpActivityFields.java

```diff
@@ -103,22 +106,25 @@
 
     //protected String govAgreementNumber;
 
-    @Interchangeable(fieldTitle = ActivityFieldsConstants.AMP_ACTIVITY_ID, importable = false)
+    @Interchangeable(fieldTitle = ActivityFieldsConstants.AMP_ACTIVITY_ID)
     @PermissibleProperty(type={Permissible.PermissibleProperty.PROPERTY_TYPE_ID})
     @VersionableFieldSimple(fieldTitle = "Internal ID", blockSingleChange = true)
     protected Long ampActivityId ;
 
-    @Interchangeable(fieldTitle = ActivityFieldsConstants.AMP_ID, required = "_ALWAYS_", importable = false)
+    @Interchangeable(fieldTitle = ActivityFieldsConstants.AMP_ID)
```

Comment:
Is this correct? Field `ampId` is no longer required?

Reply:
required is not used when importable is false